core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -395,6 +398,8 @@ object KafkaConfig {
   val LogCleanerDeleteRetentionMsProp = "log.cleaner.delete.retention.ms"
   val LogCleanerMinCompactionLagMsProp = "log.cleaner.min.compaction.lag.ms"
   val LogCleanerMaxCompactionLagMsProp = "log.cleaner.max.compaction.lag.ms"
+  val LogCleanerCompactionStrategyProp = "log.cleaner.compaction.strategy"
```

Comment:
Similarly we can define this value as "log.cleaner." + TopicConfig.COMPACTION_STRATEGY_CONFIG.